src/utils/permissions.ts

```diff
@@ -104,87 +105,77 @@ export async function hasChannelPermissions(
   const member = guild.members.get(memberID);
   if (!member) return false;
 
-  const memberOverwrite = channel.permission_overwrites?.find((o) =>
-    o.id === memberID
-  );
-
-  const rolesOverwrites = channel.permission_overwrites?.filter((o) =>
-    member.roles.includes(o.id)
-  );
-
-  const everyoneOverwrite = channel.permission_overwrites?.find((o) =>
-    o.id === guild.id
-  );
+  let memberOverwrite: RawOverwrite | undefined = undefined;
+  let everyoneOverwrite: RawOverwrite | undefined = undefined;
```

Comment:
Variables are initialized with the value `undefined` if not a value is not assigned.
```ts
let everyoneOverwrite: RawOverwrite | undefined; 
```